src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs

```diff
@@ -230,6 +230,8 @@ protected internal void UpdateWriteMetrics()
             Metrics.OtherDbWrites++;
     }
 
+    public long GetSize() => long.Parse(_db.GetProperty("rocksdb.total-sst-files-size"));
```

Comment:
How long does it take to do 1000 of such gets while DB is being modified?

Reply:
I didn't measure. But it's executed once per 5 seconds. So it shouldn't affect overall performance.